packages/ra-ui-materialui/src/list/EmptyState.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Inbox from '@material-ui/icons/Inbox';
+import { makeStyles } from '@material-ui/styles';
+import { useTranslate } from 'ra-core';
+import { CreateButton } from '../button';
+import inflection from 'inflection';
+
+const useStyles = makeStyles({
+    message: {
+        textAlign: 'center',
+        fontFamily: 'Roboto, sans-serif',
+        opacity: 0.5,
+        margin: '0 1em',
+    },
+    icon: {
+        width: '9em',
+        height: '9em',
+    },
+    toolbar: {
+        textAlign: 'center',
+        marginTop: '2em',
+    },
+});
+
+const EmptyState = ({ resource, basePath }) => {
+    const classes = useStyles();
+    const translate = useTranslate();
+
+    const resourceName = translate(`resources.${resource}.name`, {
+        smart_count: 0,
+        _: inflection.humanize(resource),
+    }).toLowerCase();
```

Comment:
We don't use `toLowerCase` anywhere else on the naming, and I `humanize` should do the trick.